strongbox-data-service/src/main/java/org/carlspring/strongbox/config/janusgraph/EmbeddedDbServerConfiguration.java

```diff
@@ -133,9 +130,10 @@ JanusGraphConfiguration janusGraphConfiguration()
     }
 
     @Bean
-    CassandraEmbeddedConfiguration cassandraEmbeddedConfiguration(JanusGraphConfiguration janusGraphConfiguration)
+    CassandraEmbeddedConfiguration cassandraEmbeddedConfiguration(@Value("${strongbox.db.janus-graph.storage-root}") String storageRoot,
```

Comment:
This is a very strange name for a property. Could you please rename it to `strongbox.db.janusgraph.storage.root`? Thanks!